docs/streaming/sources/kafka.mdx

```diff
@@ -14,8 +14,30 @@ include_metadata: false
 api_version: "0.10.2"
 batch_size: 100
 timeout_ms: 500
+auto_offset_reset: latest # Optional
+max_partition_fetch_bytes: 1048576 # Optional
+offset: None # Optional
```

Comment:
You can put these fields all under offset as a dictionary
```
offset:
  type:
  value:
  partitions: 
```

Reply:
Modified in 9b890bb8c9df6a8549ce684c5edf72273c42e2e2